mypy_django_plugin/main.py

```diff
@@ -302,7 +324,12 @@ def get_type_analyze_hook(self, fullname: str) -> Optional[Callable[[AnalyzeType
             return partial(handle_annotated_type, django_context=self.django_context)
 
     def get_dynamic_class_hook(self, fullname: str) -> Optional[Callable[[DynamicClassDefContext], None]]:
-        # Create a new manager class definition when a manager's '.from_queryset' classmethod is called
+        """
+        Create a new manager class from manager's `.from_queryset` method
+
+        Create a new manager class definition when a manager's `.from_queryset`
+        classmethod is called
```

Comment:
We should keep 1 of these phrases

```suggestion
        Create a new manager class definition when a manager's `.from_queryset`
        classmethod is called
```